custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -77,9 +77,9 @@ def load_custom_hooks(self):
                 "Failed loading hooks from [{}] : {}".format(custom_file_path, e)
             )
 
-    def load_model_from_artifact(self):
+    def load_model(self):
```

Comment:
renaming to reflect what the method is doing: load either via hook or using model artifact